### ISSUE TITLE
publish app code

### ISSUE THREAD

**Issue by afeld** [cuny-nytech/jan-teamC#5]
Get it in the repo!

**Comment by sdjukic** [comment]
First one is done.
I should be able to finish what I think is necessary for the code to be put on Heroku as alpha. I created initial admin user in seed.rb, since this info is public I have to implement password change for users in edit action (I can change the name already), later on I will allow email change but that is not that important. After that only wrinkle to fix is avatar image upload and edit. I had avatar image upload before when user is created, but now I want to move that code into edit.

What kind of instructions do you mean (like execute bundle install; bundle exec rake db:migrate; bundle exec rake db:seed ... finished by rails s)?

**Comment by afeld** [comment]
> What kind of instructions do you mean (like execute bundle install; bundle exec rake db:migrate; bundle exec rake db:seed ... finished by rails s)?

Yep! Can be as simple as listing those commands, if that's all it takes. Whatever steps are required to take the app from zero to running.

**Comment by afeld** [comment]
> later on I will allow email change but that is not that important. After that only wrinkle to fix is avatar image upload and edit. I had avatar image upload before when user is created, but now I want to move that code into edit.

You should make Issues for the outstanding TODOs so everyone keeps updated :wink:

**Comment by sdjukic** [comment]
I will.
Thanks, for the suggestions, it's good to learn git workflow in a group project. It turns out that I do have ability to change password (my initial guess how it should work was good).

What do you think of creating initial admin user in seed.rb? What would you recommend for such case, since we are not allowing creating admins via REST interface.

**Comment by afeld** [comment]
> it's good to learn git workflow in a group project

:+1: Yeah, takes Git/GitHub use to another level.

> What do you think of creating initial admin user in seed.rb?

Sounds reasonable to me!

**Comment by sdjukic** [comment]
Updated README.
